### PR TITLE
Fix for Issue 283

### PR DESCRIPTION
**Pull request by metacosm**
Fix for #283: Implemented cycling through duration and remaining time for right label in control bar.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -12,4 +12,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 1660afcf8e29f4d915d2c02e6adc97cd9068a0dd
 
-COCOAPODS: 1.2.0.beta.3
+COCOAPODS: 1.2.0.rc.1
```
Comment (lhc70000) [inline Podfile.lock]: Please discard changes to this file.

File: iina/DurationDisplayView.swift [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+//  DurationDisplayView.swift
+//  iina
+//
+//  Created by Christophe Laprun on 26/01/2017.
+//  Copyright © 2017 lhc. All rights reserved.
+//
+
+import Foundation
+
+class DurationDisplayView: NSTextField {
```
Comment (lhc70000) [inline iina/DurationDisplayView.swift]: I prefer a name ended with `TextField`, like `DurationDisplayTextField`.

File: iina/MainWindowController.swift [hunk 3]
```diff
@@ -1088,9 +1088,7 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
     let percantage = (pos.second / duration.second) * 100
     leftLabel.stringValue = pos.stringRepresentation
     touchBarCurrentPosLabel?.stringValue = pos.stringRepresentation
-    if withDuration {
-      rightLabel.stringValue = duration.stringRepresentation
-    }
+    rightLabel.updateText(with: duration, given: pos)
```
Comment (lhc70000) [inline iina/MainWindowController.swift]: Why removed `if withDuration`?

File: Podfile.lock (hunk 1 above)
Comment (metacosm) [inline Podfile.lock]: Sorry, didn't mean to include it, missed it.

File: iina/DurationDisplayView.swift (hunk 2 above)
Comment (metacosm) [inline iina/DurationDisplayView.swift]: Will do.

File: iina/MainWindowController.swift (hunk 3 above)
Comment (metacosm) [inline iina/MainWindowController.swift]: Seems like when the control bar is shown on mouse hover, withDuration is set to false. This means that, when the mode is set to remaining time, if we keep the check, the label will never be updated. The other alternative would be to set withDuration to true when the control bar is shown but I think that would amount to removing the check (which is what I did). I might have missed something but I actually don't quite understand why the label is only updated when withDuration is set to true.

**Comment by lhc70000** [comment]
Resolved conflicts. Please update your local branch if necessary (with `git pull --rebase`).

File: iina/MainWindowController.swift (hunk 3 above)
Comment (lhc70000) [inline iina/MainWindowController.swift]: Oh that makes sense. Please stay at your current changes.

**Comment by metacosm** [comment]
Changes have been made. Thank you!

**Comment by lhc70000** [comment]
`Podfile.lock` is still modified...

**Pull request merged**